pkg/cephfs/controllerserver.go

```diff
@@ -311,7 +310,7 @@ func (cs *ControllerServer) ValidateVolumeCapabilities(
 	}, nil
 }
 
-// ExpandVolume expand CephFS Volumes on demand based on resizer request
+// ControllerExpandVolume expand CephFS Volumes on demand based on resizer request
```

Comment:
Grammar nit: when we're treating the first word as a noun we want the verb to agree, so "expand" needs to become "expands"